inc/spbc-settings.php

```diff
@@ -2384,13 +2384,13 @@ function spbc_field_scanner__show_accordion( $direct_call = false ) {
 	}
  
 	if($spbc->settings['scanner__outbound_links'])
-		$tables_files['outbound_links'] = __('Shows you the list of outgoing links from your website and websites on which they linking to.', 'security-malware-firewall');
+		$tables_files['outbound_links'] = __('Outgoing links from this website and websites the links are leading to found', 'security-malware-firewall');
```

Comment:
Shouldn't the "found" word lead the sentence?
To me, it looks weird at the end of the sentence

Reply:
Fixed.